crypto/pki/pki.go

```diff
@@ -178,6 +185,18 @@ func New(public, private, config string) (*PKI, error) {
 	if p.intermediateKey, err = getPath(private, "intermediate_ca_key"); err != nil {
 		return nil, err
 	}
+	if p.sshHostCert, err = getPath(public, "ssh_host_key.pub"); err != nil {
```

Comment:
wondering about consistency here. in `step ssh certificate` you use `-cert.pub` to denote certificate files. Why not do the same here? Otherwise I assume it's just a pub key.

Reply:
Gonna leave it as is for now because I don't wanna break stuff, but this is still an open question for me. I think the file names should be "..-cert.pub" since that's how I understand these keys will be used.

Reply:
They are public keys not certificates, renaming the variable to something like sshHostPubKey and sshUserPubKey might be clearer.